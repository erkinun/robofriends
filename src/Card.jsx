import React from 'react'

const card = ({ name }) => {
  const url = `https://robohash.org/${name}`
  return (
    <div>
      <img alt='photo' src={url} />
      <div>
        <h2>{name}</h2>
        <p>jane.doe@gmail.com</p>
      </div>
    </div>
  )
}

export default card
