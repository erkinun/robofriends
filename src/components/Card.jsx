import React from 'react'

const card = ({ id, name, username, email }) => {
  const url = `https://robohash.org/${id}`
  return (
    <div className='bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5'>
      <img alt='robot' src={url} />
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  )
}

export default card
