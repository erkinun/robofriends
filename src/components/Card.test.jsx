import React from 'react'
import { shallow } from 'enzyme'
import Card from './Card'

it('renders cards successfully', () => {
  expect(shallow(<Card />)).toMatchSnapshot()
})
