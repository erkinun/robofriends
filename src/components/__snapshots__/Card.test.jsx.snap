// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders cards successfully 1`] = `ShallowWrapper {}`;
